Mathlib/AlgebraicTopology/SimplicialObject/Basic.lean

```diff
@@ -245,6 +252,47 @@ def whiskering {n} (D : Type*) [Category D] : (C ⥤ D) ⥤ Truncated C n ⥤ Tr
 
 variable {C}
 
+section Meta
+
+open Mathlib.Tactic (subscriptTerm) in
```

Comment:
Please add cross-references to the other macro/delaborator, and make sure both clearly describe their scope.